src/linux-distro.ts

```diff
@@ -104,10 +104,22 @@ function listDistroIds({ id, version, codename }: { id: string, version: string,
       return [{ value: 'rhel' + version + '0', priority: 100 }];
     case 'redhatenterprise':
     case 'redhatenterpriseserver': {
+      const toRhelVersions = (v: number, i: number) => {
+        if (v === 80) {
+          return [
+            { value: 'rhel80', priority: (i + 1) * 100 },
+            { value: 'rhel8', priority: ((i + 1) * 100) + 1 }
+          ];
+        } else {
+          return [{ value: 'rhel' + v, priority: (i + 1) * 100 }];
```

Comment:
With the suggestion below, you can just return an object here in this case where it's a single item

```suggestion
          return { value: 'rhel' + v, priority: (i + 1) * 100 };
```